src/wire-protocol.ts

```diff
@@ -80,9 +80,9 @@ export type QueryInfo = {
   /** The last transaction timestamp of the query. A Unix epoch in microseconds. */
   txn_ts: number;
   /** A readable summary of any warnings or logs emitted by the query. */
-  summary?: string;
+  summary: string;
```

Comment:
Looks to me like all fields in `QueryInfo` are optional:

https://github.com/fauna/core/blob/master/ext/api/src/main/resources/openapi.yml#L353